model/registry/azure.yaml

```diff
@@ -0,0 +1,115 @@
+groups:
+  - id: registry.azure.sdk
+    prefix: az
+    type: attribute_group
+    brief: >
+      This document defines generic attributes for Azure SDK.
+    attributes:
+      - id: service_request_id
+        type: string
+        stability: experimental
+        brief: "The unique reference string that identifies a request"
+        examples:
+          - "00000000-0000-0000-0000-000000000000"
+  - id: az.log
+    stability: experimental
+    type: event
+    name: az.log
+    brief: >
+      This document defines generic attributes for Azure Logs, see [Azure Log Schema](https://learn.microsoft.com/azure/azure-monitor/essentials/resource-logs-schema) for more details.
+    attributes:
+      - ref: event.name
+      - ref: az.service_request_id
+      - ref: cloud.resource_id
+    # Future Note: When the build tools support this definition please uncomment and validate the details
+    # included here and what has been added to the manual markdown table
+    # body:
+    #   fields:
+    #     - id: category
+    #       type: string
+    #       stability: experimental
+    #       brief: "The Azure category of the log entry"
+    #       examples:
+    #         - AuditEvent
+    #         - GatewayLogs
+    #         - ApplicationGatewayAccessLog
+    #     - id: correlation.id
+    #       type: string
+    #       stability: experimental
+    #       brief: "The correlation ID of the log entry"
+    #       examples:
+    #         - 607964b6-41a5-4e24-a5db-db7aab3b9b34
+    #     - id: duration
+    #       type: int
+    #       stability: experimental
+    #       brief: "The duration of the operations in milliseconds"
+    #       examples:
+    #         - 1000
+    #     - id: identity
+    #       type: string
+    #       stability: experimental
+    #       brief: >
+    #         "A JSON blob that describes the identity of the user or application that performed the operation."
+    #         "Typically, this field includes the authorization and claims or JWT token from Active Directory."
+    #         "Warning: this field contains sensitive PII information."
+    #       examples:
+    #         - "someone"
+    #     - id: operation.name
+    #       type: string
+    #       stability: experimental
+    #       brief: "The name of the operation"
+    #       examples:
+    #         - SecretGet
+    #         - Microsoft.ApiManagement/GatewayLogs
+    #         - ApplicationGatewayAccess
+    #     - id: operation.version
+    #       type: string
+    #       stability: experimental
+    #       brief: "The version of the operation"
+    #       examples:
+    #         - "1.0"
+    #     - id: properties
+    #       type: template[string[]]
+    #       stability: experimental
+    #       brief: "The properties of the log entry"
+    #       examples:
+    #         - "{'string': 'value'}"
```

Comment:
please use example format as in https://github.com/open-telemetry/semantic-conventions/blob/d7115c8aecbfaa6ed2f5d57def9761bca3364af2/model/registry/http.yaml#L138